Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -61,7 +61,7 @@ struct TestCommand: AsyncParsableCommand {
     
     @Option(
         name: .long,
-        help: "Tests will run <number> of times until they succeed."
+        help: "Tests will retry <number> of times until they succeed."
```

Comment:
```suggestion
        help: "Tests will retry <number> of times until they succeed. For example, if 1 is specified, the test will be retried at most once, hence it will run up to 2 times"
```